src/saveload/saveload.cpp

```diff
@@ -578,7 +578,7 @@ static inline uint SlGetArrayLength(size_t length)
  * @param conv VarType type of variable that is used for calculating the size
  * @return Return the size of this type in bytes
  */
-static inline uint SlCalcConvMemLen(VarType conv)
+uint SlCalcConvMemLen(VarType conv)
```

Comment:
In #9335 I revert this back to a `static inline`, but for my sanity (and yours) in this PR I first make it available for `settings.cpp`.